tests/text/test_wer.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from torchmetrics.text.wer import WER
+
+
+@pytest.mark.parametrize(
+    "hyp,ref,score",
+    [("hello world", "hello world", 0.0), ("hello world", "Firwww", 1.0)],
+)
+def test_wer_same(hyp, ref, score):
```

Comment:
can we test directly against `jiwer` here? to make sure that our wrapping does not introduce any bugs

Reply:
I have added the `jiwer` test too please take a look

Reply:
@SkafteNicki meant compare score computed by `jiwer` and our implementation

Reply:
A simple change @gagan3012, using the `jiwer` library, calculate the WER with the same `hyp` and `ref`, ensuring it matches the same score you compute via `metric.compute()`.

Reply:
```python
def test_wer(hyp, ref):
    metric = WER()
    metric.update(hyp, ref)
    assert metric.compute() == compute_measures(ref, hyp)['wer']
```
Is this the right way @Borda @SkafteNicki @SeanNaren